src/core/packages/notifications/browser/index.ts

```diff
@@ -14,6 +14,8 @@ export type {
   ToastInput,
   IToasts,
   ToastInputFields,
+  NotificationCoordinator,
+  NotificationCoordinatorState,
```

Comment:
This type also depends on `NotificationCoordinatorPublicApi`

Reply:
Not sure I follow here, we already export `NotificationCoordinatorPublicApi ` and it's value can be inferred by evaluating the return type of `NotificationCoordinator`

Reply:
Our docs system refuses to document interfaces/types that depend on other types that are not exported.
If we are exporting `NotificationCoordinator`, and its definition references `NotificationCoordinatorPublicApi`, we need to export both in the `index.ts` files for our docs system to export the documentation of `NotificationCoordinator`.